File: Client/typescript/module/user/UserModel.ts
module FifaLeagueClient.Module.User {
    export class UserModel {

        public Id:string;
        public Name: string;
        public Password: string;
        public ConfirmPassword: string;
        public AdministratorRole: boolean;


        // build the model directly from the data returned by the service
        constructor(data){
            if(data != null){
                this.Id = data.Id;
                this.Name = data.Name;
                this.Password = data.Password;
                this.ConfirmPassword = data.ConfirmPassword;
                this.AdministratorRole = data.AdministratorRole;
            }
        }
    }
}

File: Server/FIFA.Server/Models/User/UserModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace FIFA.Server.Models
{
    // Model representing an user
    public class UserModel
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Password { get; set; }
        public bool AdministratorRole { get; set; }
    }
}